cmd/dnet/dnet.go

```diff
@@ -192,10 +192,13 @@ func createDefaultNetwork(c libnetwork.NetworkController) {
 				ipamOption(nw))
 		}
 
-		if _, err := c.NetworkByName(nw); err == nil {
-			logrus.Debugf("Default network %s already present", nw)
-			return
+		if n, err := c.NetworkByName(nw); err == nil {
+			logrus.Debugf("Default network %s already present. Deleting it", nw)
```

Comment:
being a default network, can you please set the `DefaultBridge` option to true so that it works just like docker0 and we also exercise bridge driver changes.